src/containers/createEvent/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { useHistory } from 'react-router-dom';
+import { ContentContainer } from '../../components';
+import {
+  AsyncRequest,
+  AsyncRequestCompleted,
+  AsyncRequestFailed,
+  asyncRequestIsComplete,
+  AsyncRequestLoading,
+  AsyncRequestNotStarted,
+} from '../../utils/asyncRequest';
+import EventsForm from '../../components/EventsForm';
+import FormInitialText from '../../components/FormInitialText';
+import { Typography } from 'antd';
+import { EventsFormData } from '../../components/EventsForm';
+import { encodeProfileFieldFile } from '../../utils/fileEncoding';
+import { EventInformation } from '../upcoming-events/ducks/types';
+import protectedApiClient from '../../api/protectedApiClient';
+
+const { Title } = Typography;
+
+const CreateEventContainer: React.FC = () => {
+  const history = useHistory();
+  const [createEventRequest, setCreateEventRequest] = useState<
+    AsyncRequest<EventInformation, any>
```

Comment:
We should start using string instead of any, honestly not sure why we didn't just do that all along but oh well live and learn